.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: node_js
 notifications:
   email: false
+dist: focal
```

Comment:
Is this change really needed? I can see other apps migrated to fec without changing this option like patchman https://github.com/RedHatInsights/patchman-ui/blob/master/.travis.yml

Reply:
It is needed for node `v18` otherwise it won't be able to install this version due to missing glibc libraries.